admin.py

```diff
@@ -25,3 +37,55 @@ def decorate(self, *args, **kwargs):
 
   return decorate
 
+
+def RequireAppAndDomainAdmin(oauth_decorator):
+  """Decorator to require the user to be an admin."""
+  def Wrap(func):
+    """Decorator to require the user to be an admin."""
+    def decorate(self, *args, **kwargs):
+      """Actual decorate function that requires admin.
+
+      Args:
+        args: Parameters passed on to the specified function if successful.
+        kwargs: Parameters passed on to the specified function if successful.
+      """
+      user = users.get_current_user()
+      if not user or user is None:
```

Comment:
Can this chuck of code be pulled out into something like below, so that it can be reused by both of the require admin methods?

def AbortIfUserIsNotLoggedIn(user):
     if not user:
       logging.error('user is not logged in')	
       self.abort(403)